Core/ALife.Core/Utility/IOHelpers.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.IO;
```

Comment:
Are these used elsewhere in this checkin? could they be folded into one of the others? It seems out of place

Reply:
Not currently used in this version of things. In the V2 project they are/will be used to help with the IO work for where settings are stored and where saves are...saved, etc.

Reply:
We could technically not bring them in, but they are also a tiny thing either way, so...